cmd/kube-apiserver/app/server.go

```diff
@@ -459,27 +459,30 @@ func BuildGenericConfig(s *options.ServerRunOptions, proxyTransport *http.Transp
 		genericConfig.DisabledPostStartHooks.Insert(rbacrest.PostStartHookName)
 	}
 
-	webhookAuthResolver := func(delegate webhookconfig.AuthenticationInfoResolver) webhookconfig.AuthenticationInfoResolver {
-		return webhookconfig.AuthenticationInfoResolverFunc(func(server string) (*rest.Config, error) {
-			if server == "kubernetes.default.svc" {
```

Comment:
preserve this in `ClientConfigForFunc`:
```
ClientConfigForFunc: func(server string) (*rest.Config, error) {
  if server == "kubernetes.default.svc" {
    return genericConfig.LoopbackClientConfig, nil
  }
  return delegate.ClientConfigFor(server)
}
```

Reply:
I think the server will be `[host|ip]:port` in `ClientConfigForFunc` so this would never be true?

Reply:
`https://kubernetes.default.svc` is a valid URL that would parse to that host. Just wanted consistency between the two

Reply:
You're right. I thought the port is required in the URL but it's not.